Rakefile

```diff
@@ -1,28 +1,37 @@
 #!/usr/bin/env rake
-require 'bundler/gem_tasks'
-require 'rspec/core/rake_task'
-require 'net/http'
-require 'net/https'
+# frozen_string_literal: true
+require "bundler/gem_tasks"
+require "rspec/core/rake_task"
+require "net/http"
+require "net/https"
 
 desc "Run RSpec"
+# RSpec::Core::RakeTask.new(:spec)
```

Comment:
Oops! 🍌  I was trying to make running `bundle exec rake` run both rspec and rubocop and left this by accident

btw, I think we could probably move `--format progress` to `.rspec` and remove the block here.